python/cucim/src/cucim/skimage/feature/tests/test_blob.py

```diff
@@ -0,0 +1,560 @@
+import math
+
+import cupy as cp
+import pytest
+from cupy.testing import assert_almost_equal
+
+from cucim.skimage import feature
+from cucim.skimage.draw import disk
+from cucim.skimage.draw.draw3d import ellipsoid
+from cucim.skimage.feature import blob_dog, blob_doh, blob_log
+from cucim.skimage.feature.blob import _blob_overlap
+
+
+@pytest.mark.parametrize(
```

Comment:
add convenience wrapper to give CuPy output for ellipsoid
```suggestion
def ellipsoid(a, b, c):
    return cp.asarray(ellipsoid_cpu(a, b, c))


@pytest.mark.parametrize(
```